src/os/ui/list.js

```diff
@@ -328,25 +350,33 @@ os.ui.ListCtrl.prototype.onChange = function(evt) {
  */
 os.ui.ListCtrl.prototype.update_ = function() {
   // Always attempt to get the updated list of items, or use the scope list of items if it doesnt exist
-  var items = os.ui.list.get(this.id || '') || this.items;
+  const items = os.ui.list.get(this.id || '') || this.items;
 
-  var prefix = this.prefix || '';
-  var suffix = this.suffix || '';
+  const prefix = this.prefix || '';
+  const suffix = this.suffix || '';
 
   if (items) {
-    for (var i = 0, n = items.length; i < n; i++) {
-      var item = items[i];
+    let i = 0;
+    const n = items.length;
+    for (; i < n; i++) {
```

Comment:
why not define i on this line?

Reply:
Quite honestly, my IDE made the recommendation and I accepted it. I benchmarked it, and it seems to be marginally faster but significantly uglier.